frontend/src/App.tsx

```diff
@@ -1,11 +1,37 @@
+import {
+  KumbhSans_500Medium,
+  KumbhSans_600SemiBold,
+} from "@expo-google-fonts/kumbh-sans";
+import { NotoSans_400Regular } from "@expo-google-fonts/noto-sans";
 import { DefaultTheme, NavigationContainer } from "@react-navigation/native";
+import { useFonts } from "expo-font";
+import * as SplashScreen from "expo-splash-screen";
+import { useCallback } from "react";
 import { View, StyleSheet } from "react-native";
 
 import Header from "./components/Header/Header";
 import AppNavigator from "./navigation/AppNavigator";
 import { linking } from "./navigation/navigationLinks";
 
+SplashScreen.preventAutoHideAsync();
+
 const App = () => {
+  const [fontsLoaded, fontError] = useFonts({
```

Comment:
how much of this (lines 19-34) was from documentation vs coded from scratch?

Reply:
everything was taken from this documentation https://docs.expo.dev/develop/user-interface/fonts/

Reply:
thanks!